dashboard/lib/pd/workshop_survey_foorm_constants.rb

```diff
@@ -0,0 +1,25 @@
+module Pd
+  module WorkshopSurveyFoormConstants
+    include SharedWorkshopConstants
```

Comment:
Do these get used at all?

Reply:
nope, I had used them then changed it, removing

Reply:
just kidding, they are used. SUBJECT_SUMMER_WORKSHOP, etc are from SharedWorkshopConstants